sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -45,6 +54,37 @@
       @pageChanged="pageChanged"
     />
   </student-page-container>
+  <v-row v-if="showNav">
+    <v-col>
+      <v-btn
+        color="primary"
+        v-show="!isFirstPage"
+        variant="outlined"
+        data-cy="previousSection"
+        @click="wizardGoNext"
+        >Start your application</v-btn
+      >
+    </v-col>
+    <v-col>
+      <v-btn
+        class="float-right"
+        color="primary"
+        v-show="!isLastPage"
+        @click="wizardGoNext"
+        >Next step</v-btn
+      >
+      <v-btn
```

Comment:
mm.. bit confused. we already have a submit btn at line 25. then why this btn?

Reply:
Removed those buttons, thanks!